modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -296,6 +303,12 @@ public ObjectDefinition putObjectDefinition(
 
 		// TODO Move logic to service
 
+		if (Validator.isNull(objectDefinition.getModifiable()) &&
```

Comment:
I think this condition is wrong, did you have a chance to test it through REST API?